dydxfoundation/bridge-user-interface
# [1/n] Add Migrate UI: set allowance + bridge tx

- add hook `useMigrateToken` that handles migrate form states, inputs, as well as transaction calling and tracking
- add ui for migrate form, with steps 1. Edit, 2. Preview, 3. Confirm (WIP)

steps to test:
1. get dv3tnt tokens and sepolia ETH (i.e. gas) https://sepoliafaucet.com/
2. connect with eth wallet, observe that dydx address is generated
3. play around with different input options (e.g. invalid amount, invalid address)
4. verify button receipt and preview step info
5. click "set allowance" and wait for button to change to "confirm migration" if allowance covers the amount you're sending
6. click "confirm migration" to move to confirmed screen (this last part is not done yet)
